x-pack/plugins/watcher/server/models/action/email_action.js

```diff
@@ -71,10 +72,18 @@ export class EmailAction extends BaseAction {
     const props = super.getPropsFromUpstreamJson(json);
 
     if (!json.actionJson.email) {
-      throw badRequest('json argument must contain an actionJson.email property');
+      throw badRequest(
+        i18n.translate('xpack.watcher.models.emailAction.absenceOfActionJsonEmailPropertyBadRequestMessage', {
+          defaultMessage: 'json argument must contain an actionJson.email property',
```

Comment:
Let's place untranslatable property names into values:
```
defaultMessage: 'json argument must contain an {propertyName} property',
values: {{ propertyName: actionJson.email }}
```

Please apply it for all similar cases.